db/migrations/20180115194734_create_taggings.cr

```diff
@@ -7,7 +7,7 @@ class CreateTaggings::V20180115194734 < Avram::Migrator::Migration::V1
     end
 
     create :taggings do
-      primary_key custom_id : Int64
+      primary_key id : Int64
```

Comment:
Why did these need to change? Does it fail if you keep them as `custom_id`? I think the idea was that we wanted to test using custom ids here. 🤔

Reply:
This was to cause failures that would be fixed when the change was made (which is why I didn't add any tests). There are other associations that test the custom primary key name. I chose to change this one because it had an association with a custom key name and was not widely used in the specs so I wouldn't have to update much

Reply:
Ah, ok. That's cool then 👍